app/views/shared/buttons/_ab_list.html.haml

```diff
@@ -70,27 +70,26 @@
     -# don't need basic info box for Unassigned button group
     %h3
       = _('Basic Information')
-    %table.style1
-      %tbody
-        %tr
-          %td.key
-            = _('Button Text')
-          %td
-            = @record.name.split('|').first
-            - display = @record.set_data.key?(:display) ? @record.set_data[:display] : true
-            = check_box_tag(display, true, display, :disabled => true)
-            &nbsp;
-            = _("Display on Button")
-        %tr
-          %td.key
-            = _('Button Hover Text')
-          %td.notwide
-            = @record.description
-        %tr
-          %td.key
-            = _('Image')
-          %td.notwide
-            %span{:class => "product product-custom-#{@record.set_data[:button_image]}"}
+    %form.form-horizontal
```

Comment:
Where are no input field, `form` tags are not really good. Use a `div` with `.form-horizontal.static` instead!